src/components/QandA/QandA.jsx

```diff
@@ -3,60 +3,118 @@ import QuestionList from './QuestionList.jsx';
 import './QAstyles/QandA.css';
 import server from '../../serverRequests.js';
 
-const QandA = ({prod}) => {
+function QandA ({prod}) {
   const [search, setSearch] = useState('');
-  const [count, setCount] = useState(1);
-  const [questions, setQuestions] = useState(['Question_1', 'Question_2', 'Question_3', 'Question_4'])
+  const [qCount, setQCount] = useState(2);
+  const [aCount, setACount] = useState(1);
+  const [productID, setProductID] = useState('');
+  const [answersID, setAnswersID] = useState([]);
+  const [results, setResults] = useState([]);
+  const [questionID, setQuestionID] = useState('642440');
+  const [questions, setQuestions] = useState([]);
+  const [answers, setAnswers] = useState([]);
```

Comment:
Are answers being stored separately from their questions? How will you keep track of what answer is tied to which question? Something you might want to think about as you go to load live data. It looks like answers are stored within the question query results, or you can get specific answers by a questionID. Might make more sense to keep the answers tied within each question as state, or handle it as each question being a react component where you can break down the question and its answers and other details into state.